web/src/scripts/filetree/contextMenu.js

```diff
@@ -44,11 +45,12 @@ const buildFileMenu = (dispatch, node) => {
     new MenuItem({
       label: 'Rename',
       click: () => {
+        const { rootPath } = controller.tree
```

Comment:
I would prefer to get `rootPath` from the redux store, since otherwise the filetree API (which isn't very good yet) will start leaking into places other than just `ProjectNagivator`. Can we pass this through from `ProjectNavigator` into `showContextMenu`?